src/pybind/mgr/dashboard/services/orchestrator.py

```diff
@@ -23,13 +23,13 @@ def __init__(self):
 
     def status(self):
         try:
-            status, desc = super(OrchestratorAPI, self).available()
-            logger.info("is orchestrator available: %s, %s", status, desc)
-            return dict(available=status, description=desc)
+            status, message = super(OrchestratorAPI, self).available()
+            logger.info("is orchestrator available: %s, %s", status, message)
+            return dict(available=status, message=message)
         except (RuntimeError, OrchestratorError, ImportError):
             return dict(
                 available=False,
-                description='Orchestrator is unavailable for unknown reason')
+                message='Orchestrator is unavailable for unknown reason.')
```

Comment:
We are making this failure even more 'unknown' by discarding the exception thrown... Was it a missing library, a `RuntimeError`...? Shouldn't we at least forward the exception message?

Reply:
I'm fine when we add the reason to the existing message:

```suggestion
                message='Orchestrator is unavailable for unknown reason: {}'.format(str(e)))
```

@bk201 I think this code is originated by you. What is your opinion?

Reply:
Good idea, +1 for this.